app/controllers/spots_controller.rb

```diff
@@ -37,17 +27,12 @@ def update
     else
       render 'edit'
     end
-
   end
 
 private
 
   def spot_params
-    params.require(:spot).permit(:address, :price, :end_time, :picture, :picture_cache)
+    params.require(:spot).permit(:address, :price, :end_time)
```

Comment:
Here, we need 'picture' and 'picture_cache' for storing picture in 3S. I'm gonna merge it and fix this problem later